snippets/base/admin/fields.py

```diff
@@ -20,9 +20,11 @@ def clean(self, value):
 class JEXLChoiceField(ChoiceField):
     def __init__(self, attr_name, *args, **kwargs):
         self.attr_name = attr_name
+        self.jexl = '{attr_name} == {value}'
+        self.jexl = kwargs.pop('jexl', self.jexl)
```

Comment:
Wondering if this would be simpler as:

`self.jexl = kwargs.pop('jexl', '{attr_name} == {value}')`

Reply:
I'm on the fence with this one but I think I'm gong to leave it as is